fl4health/preprocessing/warmed_up_module.py

```diff
@@ -0,0 +1,121 @@
+import json
+import os
+from logging import INFO
+from pathlib import Path
+from typing import Optional
+
+import torch
+from flwr.common.logger import log
+
+
+class WarmedUpModule:
+    """This class is used to load a pretrained model into the current model."""
+
+    def __init__(
+        self,
+        pretrained_model: Optional[torch.nn.Module] = None,
```

Comment:
Very optional, but if you want to shorten these types (`torch.nn.Module`), we have been doing
```
import torch.nn as nn
```
in other files, which allows you to type these as just `nn.Module`. It's fine as is, just wanted to let you know in case you wanted to.